workspaces/docs/docs/goldstack/configuration/aws-user-env-variables.md

```diff
@@ -22,4 +22,4 @@ The `AWS_USER_NAME` variable is optional but can be useful for explicitly refere
     AWS_DEFAULT_REGION: us-west-2
 ```
 
```

Comment:
I can't mark it directly in this PR comment, but there is a "double header" on line#1 above (`Credentials in Environment Variables`)

> [Using environment variables](https://docs.goldstack.party/docs/goldstack/configuration#using-environment-variables)
> 
> [Credentials in Environment Variables](https://docs.goldstack.party/docs/goldstack/configuration#credentials-in-environment-variables)
> 
> Goldstack can read AWS Access Key ID and Secret Access Key from environment variables. The easiest way is to set the following environment variables: